examples/js/controls/OrbitControls.js

```diff
@@ -36,7 +36,7 @@ THREE.OrbitControls = function ( object, domElement ) {
 	// How far you can orbit vertically, upper and lower limits.
 	// Range is 0 to Math.PI radians.
 	this.minPolarAngle = 0; // radians
-	this.maxPolarAngle = Math.PI; // radians
+	this.maxPolarAngle = 11 * Math.PI / 24; // radians
```

Comment:
Sorry, this is not acceptable. We can't modify `OrbitControls` to accommodate your use case.

Reply:
OK, I cancel my change in existing OrbitControls.js file in my last commit, and I applied the change in my specific example file misc_controls_earth.js.